include/toml++/toml_node_view.h

```diff
@@ -88,13 +89,14 @@ TOML_NAMESPACE_START
 			node_view(const node_view&) noexcept = default;
 
 			///// \brief	Copy-assignment operator.
-			TOML_NODISCARD_CTOR
```

Comment:
This looked like a mistake—nodiscard doesn't make much sense on an assignment operator—I figured it was meant to be on the move constructor below.

Reply:
Yup, definite copy-pasta.